src/SFA.DAS.ApplyService.InternalApi/Startup.cs

```diff
@@ -223,10 +229,7 @@ private void ConfigureDependencyInjection(IServiceCollection services)
 
             services.AddTransient<IEmailTemplateRepository, EmailTemplateRepository>();
 
-            // NOTE: These are SOAP Services. Their client interfaces are contained within the generated Proxy code.
-            services.AddTransient<CharityCommissionService.ISearchCharitiesV1SoapClient, CharityCommissionService.SearchCharitiesV1SoapClient>();
-            services.AddTransient<CharityCommissionApiClient, CharityCommissionApiClient>();
-            // End of SOAP Services
+            services.AddTransient<CharityCommissionOuterApiClient, CharityCommissionOuterApiClient>();
```

Comment:
We should be using OuterApi client here. Move the outer API client in to Infrastructure project, that way you can reference it in here and in apply web.

Reply:
This comment was not addressed. We need to move outer api to a common project instead of having two copies of the same interface and implementation. You can use Application project to add your interface in and use Infrastructure project to add your implementation.